### PR TITLE
Automatic translator updates

### PR DESCRIPTION
**Pull request by abaevbog**
Addresses [Issue # 1](https://github.com/zotero/translation-server/issues/1)
Automatic translator updates for /web requests. The logic is:
1. On load, fetch the metadata. Once it is loaded, check if there are new translators in the metadata that are not present in _cache. Load them up as well if so.
2. Once the request comes in and all relevant translators are fetched, use `updateTranslatorIfNeeded` function to check if their lastUpdated date is before the lastUpdated date in the metadata. If so, load the translator's code from the repo, update the _cache.
3. Returns updated (if applicable) translators to translate the items.

### CODE REVIEW

File: src/translators.js [hunk 1]
```diff
@@ -1,78 +1,64 @@
 /*
-    ***** BEGIN LICENSE BLOCK *****
```
Comment (dstillman) [inline src/translators.js:2]: Revert unrelated whitespace changes — `git reset` to the parent commit, use `git co -p`, and force-push

(You'll want to configure your editor not to reindent the whole file.)

**Comment by abaevbog** [comment]
Moving this to a [PR from a feature branch](https://github.com/zotero/translation-server/pull/158)
